src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,72 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React, { useState } from 'react';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [selectedTab, setToggle] = useState(tabs[0]);
+  const selectedTabId: Tab = tabs.find(
+    tab => selectedTab.id === tab.id,
+  ) || tabs[0];
+  // nst resultco = selectedTab.id === tab.id;
+  const chooseTab = (e:React.MouseEvent<HTMLAnchorElement>,
+    tab: Tab) => {
+    e.preventDefault();
+    if (selectedTab.id === tab.id) {
+      return;
+    }
+
+    // setResult(true);
+    setToggle(tab);
+  };
```

Comment:
Replace the `setToggle` call with the `onTabSelected` prop to handle tab selection.